src/javascript/app/ng-wig/ng-wig-toolbar.provider.js

```diff
@@ -11,7 +11,7 @@ angular.module('ngWig').provider('ngWigToolbar', function () {
   var defaultButtonsList = ['list1', 'list2', 'bold', 'italic', 'link'];
 
   var isButtonActive = function () {
-    return this.command && document.queryCommandState(this.command);
+    return this.command !== undefined && document.queryCommandState(this.command);
```

Comment:
During execution of the tests, `return this.command` returns **undefined**. Thus, the whole function tends to return **undefined** instead of **false**. Is it possible that missing parentheses are the problem? For example:

`return (this.command && document.queryCommandState(this.command));`